wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/ProfileHeaderViewController.swift

```diff
@@ -355,6 +380,12 @@ final class ProfileHeaderViewController: UIViewController {
         }
     }
 
+    private func updateQRCodeButton() {
+        let qrCodeAction = UIAction { _ in }
```

Comment:
We're not doing anything here but I assume this is part of another ticket.

Reply:
It's here https://github.com/wireapp/wire-ios/pull/1957